jax_rocm_plugin/jaxlib_ext/tools/build_gpu_kernels_wheel.py

```diff
@@ -142,7 +142,7 @@ def prepare_wheel_rocm(
       f"_triton.{pyext}",
       f"rocm_plugin_extension.{pyext}",
   ]
-  runpath = '$ORIGIN/../rocm/lib:$ORIGIN/../../rocm/lib'
+  runpath = '$ORIGIN/../rocm/lib:$ORIGIN/../../rocm/lib:/opt/rocm/lib'
```

Comment:
I know that in the past we've assumed that ROCm will always be installed under `/opt/rocm/`, but that isn't going to always be the case going forward. TheRock and ROCm wheels will be installed in other locations, and our wheels are going to have to handle that. Not sure if you can do it with rpath, but can you check the `ROCM_HOME` environment variable for the ROCm path?

Reply:
> I know that in the past we've assumed that ROCm will always be installed under `/opt/rocm/`, but that isn't going to always be the case going forward. TheRock and ROCm wheels will be installed in other locations, and our wheels are going to have to handle that. Not sure if you can do it with rpath, but can you check the `ROCM_HOME` environment variable for the ROCm path?

I removed --force-rpath for enabling users to be able to set LD_LIBRARY_PATH